qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -147,15 +182,28 @@ def reps(self, reps: int) -> None:
         """
         self._reps = reps
 
-    def compute_minimum_eigenvalue(
-        self,
-        operator: OperatorBase,
-        aux_operators: Optional[List[Optional[OperatorBase]]] = None,
-    ) -> MinimumEigensolverResult:
-        self._ansatz = QAOAAnsatz(
-            operator,
-            reps=self.reps,
-            initial_state=self.initial_state,
-            mixer_operator=self.mixer,
+    def _send_job(self, inputs: Dict[str, Any], options: Dict[str, Any]):
```

Comment:
Why do we need to overwrite this? Couldn't we also include the callback here? 🙂

Reply:
Solved here: 3f95de2